bin/requirements.txt

```diff
@@ -19,8 +19,10 @@ python-slugify>=2.0
 pytz
 PyYAML>=3.0
 requests
-ruff==0.0.259
+rich
+ruff~=0.3.4
 stop-words
 texsoup~=0.3.1
 tqdm
 wheel>=0.33.4
+-e python/
```

Comment:
Also, I see that I already added the local version of the Python library to the requirements.txt, so that it can be used from all scripts in the Makefile etc.; technically only necessary after switching to the new build pipeline, I guess.